packages/react-client/src/ReactClientConsoleConfigBrowser.js

```diff
@@ -65,12 +63,6 @@ export function printToConsole(
     );
   }
 
-  if (methodName === 'error' && __DEV__) {
-    error.apply(console, newArgs);
-  } else if (methodName === 'warn' && __DEV__) {
-    warn.apply(console, newArgs);
-  } else {
-    // $FlowFixMe[invalid-computed-prop]
-    console[methodName].apply(console, newArgs); // eslint-disable-line react-internal/no-production-logging
-  }
+  // $FlowFixMe[invalid-computed-prop]
+  console[methodName].apply(console, newArgs); // eslint-disable-line react-internal/no-production-logging
```

Comment:
Looking at [this change](https://react-builds.vercel.app/commits/6983e940e6ab472434ba0665d57e6068d896f3ba/files/facebook-www/ReactDOM-dev.modern.js?compare=9a22b955e59132ff2361bb6549368d07d7de2420), this could call original console methods?

Or are they guaranteed to be transformed at this point? So we end up calling same `warn` and `error` from `consoleWithStackDev`?

Reply:
They are not guaranteed to be transformed. This affects not just this but the Flight log replaying. In other words, it won't go through the warning module in www/rn. But both of these are only affecting Flight so we just need to figure that out once Flight when added to www what to do avoid Flight logs in www.